tests/components/squeezebox/test_config_flow.py

```diff
@@ -87,6 +94,45 @@ async def test_user_form(hass: HomeAssistant) -> None:
         assert len(mock_setup_entry.mock_calls) == 1
 
 
+async def test_options_form(hass: HomeAssistant) -> None:
+    """Test we can configure options."""
+    entry = MockConfigEntry(
+        data={
+            CONF_HOST: HOST,
+            CONF_PORT: PORT,
+            CONF_USERNAME: "",
+            CONF_PASSWORD: "",
+            CONF_HTTPS: False,
+        },
+        unique_id=UUID,
+        domain=DOMAIN,
+        options={CONF_BROWSE_LIMIT: 1000, CONF_VOLUME_STEP: 5},
+    )
+
+    entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(entry.entry_id)
+    await hass.async_block_till_done()
+
+    result = await hass.config_entries.options.async_init(entry.entry_id)
+
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "init"
+
+    # simulate manual input of options
+    result2 = await hass.config_entries.options.async_configure(
```

Comment:
Don't create a new variable.

```suggestion
    result = await hass.config_entries.options.async_configure(
```

Reply:
OK.  Joostlek said the same and I'll change it.  Interested in why though.  To me, having result on the left and right of the = sign makes it more difficult to read - it would be easy miss that result had been redefined while reading through it.  Anyway, I'll change it.